flutter_vlc_player_platform_interface/lib/src/method_channel/method_channel_vlc_player.dart

```diff
@@ -174,15 +167,15 @@ class MethodChannelVlcPlayer extends VlcPlayerPlatform {
 
   @override
   Future<void> setStreamUrl(
-    int textureId, {
+    int viewId, {
     required String uri,
     required DataSourceType type,
     String? package,
     bool? autoPlay,
     HwAcc? hwAcc,
   }) async {
     var message = SetMediaMessage();
-    message.textureId = textureId;
+    message.textureId = viewId;
```

Comment:
It seems to be inconsistent here now, do we need to rename here?

Reply:
No, it's fine. 
Since in class `VlcPlayerPlatform` at `vlc_player_platform_interface.dart`, all methods have an input parameter with the name `viewId`, when we implement that class we should follow the same name convention.

Reply:
The player no longer uses textures, so `textureId` is an outdated naming.
But I believe `viewId` is not specific enough, too.

What do you think about changing the name of this entity to something like `playerId` or `playerInstanceId`?

Reply:
We followed the same procedure as [video_player](https://github.com/flutter/plugins/blob/master/packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart) package and since our implementation is based on platform view, I think `viewId` is a proper name.

Reply:
I'm sorry we are not discussing `viewId` naming, but rather changing `textureId` -> `playerId`